python_ta/debug/accumulation_table.py

```diff
@@ -0,0 +1,158 @@
+"""
+Table data structure that prints a nicely formatted table
+for an accumulator loop
+"""
+from __future__ import annotations
+
+import inspect
+import sys
+
+import astroid
+import tabulate
+
+
+def no_of_whitespaces(with_line: str) -> int:
```

Comment:
Rename to `num_whitespaces`. (In general "no" is not a good short form for "number", since "no" also means... "no")